### PR TITLE
fix(overflow): "Show More" button position should be in the bottom

### PR DESCRIPTION
**Pull request by shaibo10**
### Playbuzz: "Show More" button position should be in the bottom :bug:

1. Style-fix: "Show More" button will be returned to the bottom
2. Style-fix: button cursor will be pointer

#13661

### CODE REVIEW

File: extensions/amp-playbuzz/0.1/amp-playbuzz.css [hunk 1]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-.pb-overflow {
+.pb-overflow[overflow] {
         width: 100%;
         height: 150px;
         position: absolute;
-        bottom: 0px;
+        bottom: 0;
```
Comment (aghassemi) [inline extensions/amp-playbuzz/0.1/amp-playbuzz.css]: unless I am missing something `0`  and `0px` behave the same, not sure how this impacts anything.

Comment (shaibo10) [reply extensions/amp-playbuzz/0.1/amp-playbuzz.css]: right it's the same

File: extensions/amp-playbuzz/0.1/amp-playbuzz.css [hunk 2]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-.pb-overflow {
+.pb-overflow[overflow] {
```
[the remark this answers is not in the document]
Comment (shaibo10) [reply extensions/amp-playbuzz/0.1/amp-playbuzz.css]: I know,
I need a stronger CSS selector then ".i-amphtml-element>[overflow] selector"
for setting the .pb-overflow div to position absolute

[the remark this answers is not in the document]
Comment (aghassemi) [reply extensions/amp-playbuzz/0.1/amp-playbuzz.css]: I see.

**Comment by aghassemi** [comment]
Thanks @shaibo10, merged.

**Comment by shaibo10** [comment]
👍

**Pull request merged**